build/inline_acceleration/firewall/scripts/install_p4-ovs.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/bash
```

Comment:
This entire file seems like it copies functions and ideas from [here](https://github.com/ipdk-io/ipdk/blob/main/build/networking/start_p4ovs.sh), [here](https://github.com/ipdk-io/ipdk/blob/main/build/networking/scripts/get_p4ovs_repo.sh), and [here](https://github.com/ipdk-io/ipdk/blob/main/build/networking/scripts/build_p4sde.sh) (maybe even additional places. It feels to me like you should instead move any shared build scripts out of the `networking` folder and into the `build/scripts` directory as a separate PR, get the build working again, and then use those in this PR.

Reply:
@mestery , the main reason we did like this is to put our patches against a specific version of the P4-OVS. You may have noticed that we used a SHA for the repos to build the system and our solution works on those SHA. 
Now if we use the scripts from networking folder, those will use main branch by default and there is no way to to guarantee the working recipe. So, after few months customer can pull the script and will see the recipe is not working.

Another point is those scripts was created for container environment where there is a hardcoded script path is /root/scripts. This is a minor point though.

Do you still want to use the exiting scripts from networking folder?

Reply:
@a37a32 I do understand the feeling that a more "guaranteed working" version of everything is needed but I think the whole IPDK (networking) environment is not at that point yet, at least not until the TAI/TDI and several targets are more stable and implemented within IPDK core + a beta/upstream candidate of p4-ovs is adapted to that. To solve this for the short term I think we must implement a more thorough CI that executes some continuous litmus test on the basic storage & networking builds to signal non working changes. That's something I for sure want to work on.

Regarding this PR I agree with @mestery that it needs to be separated in a change on the basic networking build (if needed!) and an "ipdk application" part that uses the basic networking build.

On that 2nd part I have one question: This PR is using a VM with a running VPP application that uses vfio-pci to connect to the virtual vhost-user interface. Is this application also capable to connect directly to a vhost-user interface (by running directly on the host, in the same net namespace or a child net-ns)? Don't know how much work implementing vhost-user support is but VPP is using DPDK am I right? By doing that we can build the app and run it against p4-ovs in CI without using VM's. That would simplify a CI implementation on the current CI environment (Github Actions) a lot for now!

Reply:
BTW: It would be great if we would also have a separate recipe within IPDK for building pktgen or any other open source packet generator (separated from this firewall recipe) that could also be used in testing... 🤔

Reply:
We have modified the script according to your suggestion. Please have a look.